ext/vscode/docs/commands.md

```diff
@@ -0,0 +1,70 @@
+# VS Code Extension Commands
```

Comment:
IMO this is a weak title, not specific enough to the fact that this is the AZD extension